### PR TITLE
Updated logic to only include passed tests

### PR DESCRIPTION
**Pull request by simontindallbjss**
## Description

- Minor change to only include passed tests
- Hardcoded 1 (pass) for TestStatus as logic now only includes passed tests
- Added IssueDateTime field which is required to filter and sort date for TFL output

Related issue: [CB2-5043](https://dvsa.atlassian.net/browse/CB2-5043)

## Before submitting (or marking as "ready for review")

- [ ] Does the pull request title follow the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) specification?
- [X] Have you performed a self-review of the code
- [ ] Have you have added tests that prove the fix or feature is effective and working
- [ ] Did you make sure to update any documentation relating to this change?

### CODE REVIEW

File: sql/10001_tfl_views.sql [hunk 1]
```diff
@@ -6,10 +6,7 @@ SELECT
     v.vin     as VIN,
     tr.certificateNumber as SerialNumberOfCertificate,
     IFNULL(fe.modTypeCode,"") as CertificationModificationType,
-    CASE SUBSTR(tr.certificateNumber,1,2)
-        WHEN 'LP' THEN 1
-        WHEN 'LF' THEN 2
-    END AS TestStatus,
+    1 AS TestStatus,
```
Comment (simontindallbjss) [inline sql/10001_tfl_views.sql:9]: Removed CASE statement as only bringing through passed tests

File: sql/10001_tfl_views.sql [hunk 2]
```diff
@@ -52,7 +49,17 @@ SELECT
...
+    CASE
+        WHEN tr.createdAt IS NOT NULL 
+            THEN tr.createdAt
+        WHEN tr.createdAt IS NULL AND tr.testTypeStartTimestamp IS NOT NULL 
+            THEN tr.testTypeStartTimestamp
+        WHEN tr.createdAt IS NULL AND tr.testTypeStartTimestamp IS NULL AND tr.testTypeEndTimestamp IS NOT NULL
+            THEN tr.testTypeEndTimestamp
+        ELSE
+            NULL
+    END AS IssueDateTime
```
Comment (simontindallbjss) [inline sql/10001_tfl_views.sql:62]: Added field for IssueDateTime to allow for filtering and sorting by VTM
Created using same logic as IssueDate, but removed formatting.

File: sql/10001_tfl_views.sql [hunk 3]
```diff
@@ -68,5 +75,5 @@ JOIN
     CVSNOP.fuel_emission fe
     ON (fe.id = tr.fuel_emission_id)
 WHERE
-    SUBSTR(tr.certificateNumber,1,2) IN ('LP', 'LF')
+    SUBSTR(tr.certificateNumber,1,2) = 'LP'
```
Comment (simontindallbjss) [inline sql/10001_tfl_views.sql:78]: Updated to only include passed tests

**Comment by ChrisPeska-BJSS** [comment]
fyi, @vanithaudt left [PR](https://github.com/dvsa/cvs-auto-svc-vott/pull/22) with changes for VOTT regression pack. There is likelihood those changes may break some of tests V. implemented.

**Review by reisedawson — APPROVED** [review]
Changes make sense to me. Happy to approve.

@ChrisPeska-BJSS Shall we consider the VOTT pack impact at a later date? VTM release to INT is tomorrow. I don't see the VOTT PR going into master before then 😅

**Comment by ChrisPeska-BJSS** [comment]
Sure 🤠 (laters)

**Review by ChrisPeska-BJSS — APPROVED** [review]

**Pull request merged**